### PR TITLE
Create pavanh007

### PR DESCRIPTION
**Pull request by pavanh007**
Required changes are updated: PR #6714

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6738"></a>

### CODE REVIEW

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @pavanh007, thank you for contributing :tada: 
Please rename your file to **`data/pavanh007.json`** :+1:

**Review by HassanTanveer — DISMISSED** [review]

**Comment by pavanh007** [comment]
> Hey @pavanh007, thank you for contributing 🎉 Please rename your file to **`data/pavanh007.json`** 👍

can you suggest me

**Comment by SurajPratap10** [comment]
> > Hey @pavanh007, thank you for contributing 🎉 Please rename your file to **`data/pavanh007.json`** 👍
> 
> can you suggest me

There's a `data` folder in the repo where every user's data is kept, you have to put your file under that folder.

**Comment by pavanh007** [comment]
> > > Hey @pavanh007, thank you for contributing 🎉 Please rename your file to **`data/pavanh007.json`** 👍
> > 
> > 
> > can you suggest me
> 
> There's a `data` folder in the repo where every user's data is kept, you have to put your file under that folder.

>> As you suggest, I removed existing file and added in data folder

**Review by kumarsonsoff3 — DISMISSED** [review]
Please add the file extension `.json` in the end of your [filename here](https://github.com/pavanh007/LinkFree/edit/pavanh007-json/data/pavanh007).

Let us know if you need any help with this! :)

**Review by Vishrut19 — CHANGES_REQUESTED** [review]
Please change the filename to data/pavanh007.json

**Comment by SaraJaoude** [comment]
> Please change the filename to data/pavanh007.json

@Vishrut19 we have brought to your attention a number of times not to duplicate comments.  As we have highlighted to you this causes confusion to the author, maintainers - and can also scare off contributors. If you do not have time to be looking through historical comments then perhaps you should not be reviewing PRs.

**Comment by Vishrut19** [comment]
> > Please change the filename to data/pavanh007.json
> 
> @Vishrut19 we have brought to your attention a number of times not to duplicate comments. As we have highlighted to you this causes confusion to the author, maintainers - and can also scare off contributors. If you do not have time to be looking through historical comments then perhaps you should not be reviewing PRs.

Hey Sara! First of I am extremely Sorry for duplicate comments but I tried to make pavan understand that this is the naming convention that needs to be followed i.e. `data/<yourname>/json`. I thought putting a comment like that would make him understand about the naming convention along with the file format. Hope it makes sense.

File: data/pavanh007
Comment (Vishrut19) [inline data/pavanh007:4]: The format is incorrect. The correct format is "Backend developer | Tech enthusiast"

**Review by Vishrut19 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Hey everyone,

Thank you all for the review. Since the pull request has been open for a while, I will be merging it into a temporary branch to address any existing errors before merging it into the main branch.

**Pull request merged**
